arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -0,0 +1,1041 @@
+                        "schema": {
+                            "$ref": "#/definitions/Error"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.PowerBI/workspaceCollections": {
+            "get": {
+                "tags": [
+                    "WorkspaceCollections"
+                ],
+                "operationId": "WorkspaceCollections_listByResourceGroup",
+                "description": "The **Get Power BI Workspace Collections in Resource Group** operation retrieves all existing Power BI Workspace Collections in the specified resource group.",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "subscriptionId",
+                        "in": "path",
+                        "description": "Azure subscription id",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "resourceGroupName",
+                        "in": "path",
+                        "description": "Azure resource group",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "api-version",
+                        "in": "query",
+                        "description": "Azure API version",
+                        "required": true,
+                        "type": "string"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Get workspaces response",
+                        "schema": {
+                            "$ref": "#/definitions/WorkspaceCollectionList"
+                        }
+                    },
+                    "default": {
+                        "description": "Default Response. It will be deserialized as per the Error defintion specified in the schema. Exception will be thrown.",
+                        "schema": {
+                            "$ref": "#/definitions/Error"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.PowerBI/workspaceCollections": {
+            "get": {
+                "tags": [
+                    "WorkspaceCollections"
+                ],
+                "operationId": "WorkspaceCollections_listBySubscription",
+                "description": "The **Get Power BI Workspace Collections in Subscription** operation retrieves all existing Power BI Workspace Collections in the specified subscription.",
```

Comment:
Add "x-ms-pageable" extension. Please read the extensions doc carefully. You can set the nextLinkName to null if you do not have nextlink. However, your client will still return an IEnumerable<WorkspaceCollection>. Basically value property is flattened.